src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/RcaController.java

```diff
@@ -133,19 +133,6 @@ public RcaController(
     this.timeUnit = timeUnit;
   }
 
-  /**
-   * Starts the pollers. Each poller is a thread that checks for the state of the system that the
-   * RCA is concerned with. - RcaConfPoller: Periodically reads a config file to determine if RCA is
-   * supposed to be running or not and accordingly sets a flag. - NodeRolePoller: This checks for
-   * the change of role for an elastic search Master node. - RcaNanny: RcaConfPoller sets the flag
-   * but this thread works to start Rca or shut it down based on the flag.
-   */
-  public void startPollers() {
-    startRcaConfPoller();
```

Comment:
It might be cleaner to keep this method and only remove the `startRcaConfPoller();` if we are incorporating that functionality in the `rcaNanny()`